Does not compile due to if.. else identation errors

Here's the error log:

```sh
~/Sawteeth> cmake .
~/Sawteeth> make | nc termbin.com 9999
/boot/home/Sawteeth/Source/stAiffWindow.cpp: In member function 'void stAiffWindow::Thread()':
/boot/home/Sawteeth/Source/stAiffWindow.cpp:116:4: error: this 'if' clause does not guard... [-Werror=misleading-indentation]
    if (sbuf[c] > max) max = sbuf[c];if (sbuf[c] < min) min = sbuf[c];
    ^~
/boot/home/Sawteeth/Source/stAiffWindow.cpp:116:37: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the 'if'
    if (sbuf[c] > max) max = sbuf[c];if (sbuf[c] < min) min = sbuf[c];
                                     ^~
cc1plus: all warnings being treated as errors
make[2]: *** [Source/CMakeFiles/Sawteeth.dir/stAiffWindow.cpp.o] Error 1
make[1]: *** [Source/CMakeFiles/Sawteeth.dir/all] Error 2
make: *** [all] Error 2
```

--- a/Source/stAiffWindow.cpp
+++ b/Source/stAiffWindow.cpp
@@ -115,7 +115,6 @@ void stAiffWindow::Thread()
 			//max, min and clipping
 			if (sbuf[c] > max)
 				max = sbuf[c];
-
 			if (sbuf[c] < min)
 				min = sbuf[c];
 

--- a/Source/stAiffWindow.cpp
+++ b/Source/stAiffWindow.cpp
@@ -113,8 +113,18 @@ void stAiffWindow::Thread()
 		for (int c = 0 ; c< (2 * sbufs); c++){
 
 			//max, min and clipping
-			if (sbuf[c] > max) max = sbuf[c];if (sbuf[c] < min) min = sbuf[c];
-			if (sbuf[c] > 0.999) sbuf[c] = 0.999; else if (sbuf[c] < -0.999) sbuf[c] = -0.999;
+			if (sbuf[c] > max)
+				max = sbuf[c];
+
+			if (sbuf[c] < min)
+				min = sbuf[c];
+
+			if (sbuf[c] > 0.999) {
+				sbuf[c] = 0.999;
+			} else if (sbuf[c] < -0.999) {
+				sbuf[c] = -0.999;
+			}
+
 			int16 tmp = (int16)(30000.0 * sbuf[c]);
 			
 			obuf[c*2] =		tmp >> 8;

--- a/Source/stApp.cpp
+++ b/Source/stApp.cpp
@@ -98,7 +98,7 @@ void stApp::RefsReceived(BMessage *message)
 				stMainWindow *win = new stMainWindow(BRect(10,29,450,150),t);
 
 				status_t temp = win->InitCheck();
-				if (JNG_OK != temp){
+				if (JNG_OK != temp) {
 					char tempc[512];
 					sprintf(tempc,
 						"ERROR\n\nLoader version = %.2f < Song version = %.2f\n\nDownload newer software at:\nwww.acc.umu.se/~bedev/software/sawteeth",
@@ -111,15 +111,16 @@ void stApp::RefsReceived(BMessage *message)
 							"Not Valid",tempc,
 							"OK",0,0,B_WIDTH_AS_USUAL,B_STOP_ALERT);
 						alert->Go();
-					}else{
+					} else {
 						BAlert *alert=new BAlert(
 							"Not Valid","ERROR\n\nNot a valid sawteeth song",
 							"OK",0,0,B_WIDTH_AS_USUAL,B_STOP_ALERT);
 						alert->Go();
 					}
-				}else
+				} else {
 					win->Show();
 					win->Activate();
+				}
 			}
 		} 
 	} 

